jobs/tests/test_views.py

```diff
@@ -39,3 +41,80 @@ def test_main_page_with_empty_database(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn("No job offers yet", response.content)
         self.assertIn("No meetups yet", response.content)
+
+
+class JobsPageTests(TestCase):
+
+    def test_jobs_page_with_empty_database(self):
+        response = self.client.get(reverse('jobs:jobs'))
+        self.assertEqual(response.status_code, 200)
+        self.assertIn("There are no job offers at this moment.", response.content)
+
+    def test_jobs_page_with_job_not_ready_to_publish(self):
+        Job.objects.create(
+            title='Intern',
+            company='Google',
+            website='http://www.google.pl/about/careers/students/',
+            city='London',
+            country='GB',
+            description='description',
+            review_status=Job.OPEN,
+        )
```

Comment:
I wasn't sure whether you can use model mommy for views tests as well - all examples are for models tests